### PR TITLE
ADD JSTS Dataset for STS TASK

### PR DESCRIPTION
**Pull request by xu3kev**
## Checklist for adding MMTEB dataset

Reason for dataset addition: Add a Japanese dataset to the STS tasks category.

- [X] I have tested that the dataset runs with the `mteb` package.
- [X] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [X] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [X] `intfloat/multilingual-e5-small`
- [X] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [X] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [X] Run tests locally to make sure nothing is broken using `make test`. 
- [X] Run the formatter to format the code using `make lint`. 
- [X] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file.

### CODE REVIEW

File: mteb/tasks/Classification/jpn/WRIMEClassification.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        eval_langs=["jpn-Jpan"],
+        main_score="accuracy",
+        date=("2011-06-01", "2020-05-31"),
+        form=["written"],
+        domains=["Social"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="The dataset is available for research purposes only. Redistribution of the dataset is prohibited.",
+        socioeconomic_status="high",
+        annotations_creators="human-annotated",
+        dialect=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/jpn/WRIMEClassification.py:29]:
```suggestion
        dialect=[],
```
None means that it is not annotated, [], means annotated and there is no dialect

**Comment by KennethEnevoldsen** [comment]
@xu3kev I see that you are stacking the PRs. If you do that please edit the PRs such that the merge tree would look like:

main <- change1 <- change 1 +2  <- change 1 + 2 + 3

(otherwise we will review the same changes twice)

File: mteb/tasks/STS/jpn/JSTS.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        eval_langs=["jpn-Jpan"],
+        main_score="cosine_spearman",
+        date=("2016-01-01", "2022-12-31"),
+        form=["written"],
+        domains=["Web"],
+        task_subtypes=None,
+        license="CC BY-SA 4.0",
+        socioeconomic_status="high",
+        annotations_creators="human-annotated",
+        dialect=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/STS/jpn/JSTS.py:31]:
```suggestion
        dialect=[],
```

File: mteb/tasks/STS/jpn/JSTS.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        reference="https://aclanthology.org/2022.lrec-1.317.pdf#page=2.00",
+        type="STS",
+        category="s2s",
+        eval_splits=["validation"],
+        eval_langs=["jpn-Jpan"],
+        main_score="cosine_spearman",
+        date=("2016-01-01", "2022-12-31"),
+        form=["written"],
+        domains=["Web"],
+        task_subtypes=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/STS/jpn/JSTS.py:27]:
```suggestion
        task_subtypes=[],
```

**Review by KennethEnevoldsen — COMMENTED** [review]
Thanks for the submissions, great to see so many for Japanese. Please see above for order of merges.

File: mteb/tasks/Classification/jpn/WRIMEClassification.py (hunk 1 above)
Comment (xu3kev) [reply mteb/tasks/Classification/jpn/WRIMEClassification.py]: Fixed in #358 and merged.

File: mteb/tasks/STS/jpn/JSTS.py (hunk 2 above)
Comment (xu3kev) [reply mteb/tasks/STS/jpn/JSTS.py]: Fixed.

**Comment by xu3kev** [comment]
cleaned up and synced to the main branch. Please check again, thank you! @KennethEnevoldsen

**Comment by KennethEnevoldsen** [comment]
Perfect @xu3kev, thanks for the addition!

**Pull request merged**